croud/rest.py

```diff
@@ -107,7 +112,10 @@ def print(self, success_message: str = None, keys: List[str] = None):
 
         if self._data is None or len(self._data) == 0:
             message = success_message or "Success."
-            print_success(message)
+            if level and level == Level.INFO.value:
```

Comment:
Since `level` has a default value and is not optional, you don't need to check for it being defined.
```suggestion
            if level == Level.INFO:
```